src/annoylib.h

```diff
@@ -929,7 +952,7 @@ template<typename S, typename T, typename Distance, typename Random>
     return true;
   }
     
-  bool build(int q, char** error=NULL) {
+  bool build(int q, int n_threads=1, char** error=NULL) {
```

Comment:
might be good to default it to `-1` and if that's the case set it to the number of cpus